api/proto/teleport/legacy/types/events/events.proto

```diff
@@ -75,6 +75,9 @@ message UserMetadata {
   // AWSRoleARN is AWS IAM role user assumes when accessing AWS console.
   string AWSRoleARN = 4 [(gogoproto.jsontag) = "aws_role_arn,omitempty"];
 
+  // AzureIdentity is the Azure identity to assume when accessing Azure API.
+  string AzureIdentity = 6 [(gogoproto.jsontag) = "azure_identity,omitempty"];
```

Comment:
Agreed - in fact, I have that change already scheduled to be made in the upcoming GCP PR. Since reordering fields is a safe operation, I'll do it then.

Thanks for the review!